llama-index-integrations/embeddings/llama-index-embeddings-voyageai/llama_index/embeddings/voyageai/base.py

```diff
@@ -22,79 +19,78 @@ class VoyageEmbedding(BaseEmbedding):
             You can either specify the key here or store it as an environment variable.
     """
 
-    _model: Any = PrivateAttr()
+    client: Optional[voyageai.Client]
```

Comment:
Why do we need it here? Doesn't seem to be necessary to me?

Reply:
Right, i just modified a bit. Since `VoyageEmbedding` is a pydantic class, attributes should be defined.

Reply:
Intuitively, this shouldn't be an Optional attribute?
Is the problem that if you don't make it optional, you will have an error in init?
Perhaps we could learn from the implementation for cohere https://github.com/voyage-ai/llama_index/blob/main/llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py#L111

Reply:
Sorry, I forgot to push the modification. 
`    client: voyageai.Client = PrivateAttr(None)
`